### PR TITLE
Docs, WebHost: Update faq_en.md

### PR DESCRIPTION
**Pull request by eudaimonistic**
Reorganizing information and adding links to some of the various guides and website pages.  Even just adding the Getting Started, Supported Games, and Server Commands links seems like a hefty upgrade.  We have good resources, we should make them obvious.

I think more can probably be done here, but I already shuffled this around a lot.  Would appreciate suggestions or feedback.

### CODE REVIEW

**Comment by kindasneaki** [comment]
I would probably reorganize things some more. Maybe something like:
- What is a randomizer
- What is a multi-world
- What does multi-game mean
- Can I generate a single-player game with archipelago
- How do I get started
- What are some common terms I should know?
- What happens if a person has to leave early
- What happens if an item is placed somewhere it is impossible to get
- I want to add a game to the Archipelago randomizer. How do I do that?

**Comment by kindasneaki** [comment]
Also I think multi world and multi game are very similar, and we might not need both?

**Comment by BootsinSoots** [comment]
I think the distinction on multi-world vs multi-game adds value to someone unfamiliar with us and why we have both terms in the software description

**Comment by eudaimonistic** [comment]
The way I see it, is the FAQ page is very likely a place people would land when they first get to the website.  There are _many_ entry points to learning about this platform.  Quite a few users only know about us because of a single randomizer mod, completely unaware that we're a multiworld/multigame platform.  It helps to explain the concepts separately because they might be completely foreign to new users.

**Comment by kindasneaki** [comment]
That makes sense. Any opinions on reorganizing?

**Comment by BootsinSoots** [comment]
I do think I like your order a little better Sneaki. Smaller concepts to more specific ones

**Comment by eudaimonistic** [comment]
> That makes sense. Any opinions on reorganizing?

That organization makes way more sense.  I arranged as suggested, and elaborated a little more on single player while doing a pass.

File: WebHostLib/static/assets/faq/faq_en.md [hunk 1]
```diff
@@ -27,24 +20,25 @@ item will be sent to player B's world over the internet.
...
-the website is not required to generate them.
+Yes. All of our supported games can be generated as single-player experiences both on the website and through installing 
+the Archipelago generator software. The fastest way to do this is on the website. Find the Supported Game you wish to play, 
+open the Settings Page, pick your settings, and click Generate Game.
 
 ## How do I get started?
 
+We have a [Getting Started](https://archipelago.gg/tutorial/Archipelago/setup/en) guide that will help you get the
+software set up.  You can use that guide to learn how to generate multiworlds.  There are also basic instructions for
+including multiple games, and hosting these multiworlds on the website for ease and convenience.
```
Comment (kindasneaki) [inline WebHostLib/static/assets/faq/faq_en.md:40]:
```suggestion
including multiple games, and hosting these multi-worlds on the website for ease and convenience.
```

File: WebHostLib/static/assets/faq/faq_en.md [hunk 2]
```diff
@@ -27,24 +20,25 @@ item will be sent to player B's world over the internet.
...
-Yes. All our supported games can be generated as single-player experiences, and so long as you download the software,
-the website is not required to generate them.
+Yes. All of our supported games can be generated as single-player experiences both on the website and through installing 
+the Archipelago generator software. The fastest way to do this is on the website. Find the Supported Game you wish to play, 
+open the Settings Page, pick your settings, and click Generate Game.
 
 ## How do I get started?
 
+We have a [Getting Started](https://archipelago.gg/tutorial/Archipelago/setup/en) guide that will help you get the
+software set up.  You can use that guide to learn how to generate multiworlds.  There are also basic instructions for
```
Comment (kindasneaki) [inline WebHostLib/static/assets/faq/faq_en.md:39]:
```suggestion
software set up.  You can use that guide to learn how to generate multi-worlds.  There are also basic instructions for
```

**Review by kindasneaki — CHANGES_REQUESTED** [review]
Just 2 small nit picks

**Comment by BootsinSoots** [comment]
I think I disagree with this, only because the word is not hyphenated on the main page of the website

**Comment by kindasneaki** [comment]
it’s very inconsistent throughout the site. In this file specifically it’s all multi-world. There is another multiworld on `line 48` that I missed also

**Comment by eudaimonistic** [comment]
...Imagine making commits without reading the entire chain...

Anyway, will recommit with explicitly using multiworld as a noun.

**Comment by eudaimonistic** [comment]
I've chosen to keep the usage of multi-game hyphenated, because it's probably good to articulate it differently than multiworld for natural emphasis.  When reading multi-game as a hyphenated term, it implies a modification to the combined noun multiworld.  In this context, I think it works better that way.

**Comment by BootsinSoots** [comment]
I think that's for the best - the main page also shows it as multi-game

**Review by kindasneaki — APPROVED** [review]

**Review by nicholassaylor — APPROVED** [review]
I like the new order that we have and the standardization of multiworld
All of the changes are reasonable and make the faq a more straightforward read

**Comment by eudaimonistic** [comment]
I've reworked the section about leaving early and turned it into a basic explanation of sync/async, so as to clarify the general flow of a multiworld.  This comes up constantly in the Discord and it seems necessary to explain here.  While I can foresee more improvements being possible beyond this point, we seem to be growing fast enough to warrant pushing this through and revisiting the page at a later date if necessary.

**Comment by black-sliver** [comment]
@LegendaryLinux as the original author, I think.

**Comment by eudaimonistic** [comment]
Better English than mine prevails.  I approve of those changes.

**Review by LegendaryLinux — APPROVED** [review]

**Pull request merged**